trait-representation-migration/remove_background_trait_links.py

```diff
@@ -0,0 +1,213 @@
+# Activate Python venv for the script - uncomment to run script on commandline
+# activate_this_file = "/path/to/bin/activate_this.py"
+# execfile(activate_this_file, dict(__file__ = activate_this_file))
+
+import cx_Oracle
+import contextlib
+import argparse
+import sys
+from tqdm import tqdm
+import csv
+import os.path
+
+sys.path.insert(0, '/path/to/gwas_data_sources')
+import gwas_data_sources
+
+import datetime
+
+
+def read_file(filename):
+    '''
+    Read file.
+
+    Args:
+        filename: Name of file provided as a commandline argument.
+
+    Returns:
+        data_map: Dictionary with the STUDY_ID as the key and a 
+        list of EFO_IDs as the dictionary value.
+    '''
+
+    with open(filename, 'r') as file:
+        lines = file.readlines()[1:]
+
+    return lines
+
+
+def process_file_contents(file_data, efo_map, cursor):
+    '''
+    Extract fields of interest from the file and clean-up values.
+
+    Args:
+        file_data: Contents of the file.
+    '''
+
+    count = 0
+
+    for line in file_data:
+        formatted_line = line.split('\t')
+
+        study_accession = formatted_line[2].strip()
+        study_id = _get_study_id(study_accession)
+
+        background_column = formatted_line[4].strip()
+
+        delimiter = '||'
+
+        if not background_column == '':
+            count += 1
+            # print('\nCount: {} StudyID: {} Accession: {}'.format(count, study_id, study_accession))
+            
+            if delimiter in background_column:
+                background_column = background_column.split(delimiter)
+
+                for background_trait in background_column:
+                    # Get the EFO_ID
+                    background_trait_id = efo_map[background_trait.strip().lower()]
```

Comment:
Just a suggestion: given the file is (more or less) manually created, I would add a test to check if the trait name is in the hash. But that's not particularly important.